springboard_iogt/static/css/super.css

```diff
@@ -0,0 +1,84 @@
+    }
+
+    #content .h1 .back{
+        padding: 1em 3em;
+        background: 20px 20px no-repeat;
+        background-size: 15px 15px;
+        background-image: url("img/arrow_red.png");
+    }
+    
+    #content .articles .barefootlaw .h1 .back{background-image: url("img/arrow_green_flipped.png")}
+    #content .articles .ebola .h1 .back{background-image: url("img/arrow_yellow_flipped.png");}
+    #content .articles .ffl .h1 .back{background-image: url("img/arrow_turquoise_flipped.png");}
+    #content .articles .mariestopes .h1 .back{background-image: url("img/arrow_red_flipped.png");}
+    #content .articles .straighttalk .h1 .back{background-image: url("img/arrow_purple_flipped.png");}
+    #content .articles .ureport .h1 .back{background-image: url("img/arrow_orange.png");}
+
+    #submenu li { 
+        padding: 0; 
+        background: #ededed; 
+        border-bottom: solid 1px #fff; }
+    #submenu li a {
+        padding: 1em 3em;
+        color: #666;
+        background: transparent 20px 20px no-repeat;
+        background-size: 15px 15px;
+    }
+    #submenu p{ 
+        display: block; 
+        font-weight: 100;
+        margin: 0;
+        font-size: 14px;
+    }
+
+    #submenu .barefootlaw a{background-image: url("img/arrow_green.png")}
+    #submenu .ebola a{background-image: url("img/arrow_yellow.png");}
+    #submenu .ffl a{background-image: url("img/arrow_turquoise.png");}
+    #submenu .mariestopes a{background-image: url("img/arrow_red.png");}
+    #submenu .straighttalk a{background-image: url("img/arrow_purple.png");}
+    #submenu .ureport a{background-image: url("img/arrow_orange.png");}
+    
+    #content .home .articles .listing {border-top: 10px solid white; position: relative;}
+    #content .home .articles .ffl:after {background-color:#32bac2; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+    #content .home .articles .barefootlaw:after {background-color:#6ec17c; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+    #content .home .articles .mariestopes:after {background-color:#e04a5a; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+    #content .home .articles .straighttalk:after {background-color:#c977b2; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+    #content .home .articles .ureport:after {background-color:#f18054; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+    #content .home .articles .ebola:after {background-color:#fdbe61; padding:0; margin:0; display:block; content: ""; width:90%; height:10px; position: absolute;right: 0;top:-10px;}
+
+    .cta .h2 a {
+        padding: 10px;
+        color: white;
+        display: block;
+        background-color: #28B0B6
```

Comment:
It wasn't clear on the original designs, but the background colour must match that of the content section. The colours are [here](https://github.com/universalcore/springboard-iogt/blob/feature/issue-15-add-smart-styling/springboard_iogt/static/css/style.css#L83). I'm attaching a new image on the Trello card that illustrates it.